images/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "web-scrobbler-images",
+  "version": "1.0.0",
+  "description": "script for generating images for web scrobbler",
+  "main": "generate-icons.ts",
+  "type": "module",
+  "scripts": {
+    "generate": "tsx generate.ts"
```

Comment:
Should we invoke this on the release pipeline and abort if the working dir is dirty? or run on master builds?

Reply:
The point of this being separated out is that the release build pipeline no longer requires invoking canvas, so mozilla doesnt run into problems reproducing the build.
As a result, we should avoid invoking this in CI for now until theres a better solution